src/vsr/superblock.zig

```diff
@@ -879,7 +874,7 @@ pub fn SuperBlockType(comptime Storage: type) type {
             vsr_state.commit_max = update.commit_max;
             vsr_state.sync_op_min = update.sync_op_min;
             vsr_state.sync_op_max = update.sync_op_max;
-            vsr_state.sync_view = update.sync_view;
```

Comment:
Crucially, this is part of vsr_state, but not checkpoint_state. So, this is replica-local state, and we can change the rules for updating this without breaking compatibility across versions